pages/style.css

```diff
@@ -0,0 +1,153 @@
+body {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+h1 {
+    text-align: center;
+    align-items: flex-end;
+    font-size: 60px;
+    height: 100px;
+    position: relative;
+    box-sizing: border-box;
+    margin: 50px;
+} 
+
+nav {
+    letter-spacing: 3px;
+    bottom: 50px;
+    position: relative;
+    text-align: center;
+    padding: 50px;
+    font-weight: 100;
+    font-family: 'Raleway', sans-serif;
+}
+
+footer {
+	position: relative;
+	width: 100%;
+	bottom: -100px;
+	background-color: #7f86c2;
+}
+
+
+#introduction {
+    font-size: 20px;
+    padding: 300px;
+    background-image: url("../images/sunset-1.jpeg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover; 
+}
+
+#introduction p {
+    position: relative;
+    right: 150px;
+    top: 100px;
+    padding-bottom: 2.5rem;
+
+}
```

Comment:
Consider using relative units like `vw` or `vh` to allow your content to scale better to larger or smaller screens.